node/module.ts

```diff
@@ -754,9 +754,20 @@ for (const key of Object.keys(nodeMods)) {
 nativeModulePolyfill.set(
   "module",
   createNativeModule("module", {
+    _cache: Module._cache,
+    _extensions: Module._extensions,
+    _findPath: Module._findPath,
+    _initPaths: Module._initPaths,
+    _load: Module._load,
+    _nodeModulePaths: Module._nodeModulePaths,
+    _pathCache: Module._pathCache,
+    _preloadModules: Module._preloadModules,
+    _resolveFilename: Module._resolveFilename,
+    _resolveLookupPaths: Module._resolveLookupPaths,
     default: Module,
-    builtinModules: Module.builtinModules,
-    createRequire: Module.createRequire,
```

Comment:
What's the reason for deleting these two? I think these 2 are still necessary

Reply:
That was a mistake. Fixed now.